src/classes/modules/java.logging/java/util/logging/FileHandler.java

```diff
@@ -6,13 +6,13 @@
  * The Java Pathfinder core (jpf-core) platform is licensed under the
  * Apache License, Version 2.0 (the "License"); you may not use this file except
  * in compliance with the License. You may obtain a copy of the License at
- * 
- *        http://www.apache.org/licenses/LICENSE-2.0. 
```

Comment:
Please avoid changes in whitespace, as we subsequently want to merge changes to `master` to other branches (to support newer Java versions). Therefore, we want to keep only essential changes.